channels-rke2.yaml

```diff
@@ -2164,6 +2164,38 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1800
     featureVersions: *featureVersions-v1
+  - version: v1.28.14+rke2r1
+    minChannelServerVersion: v2.8.3-alpha1
+    maxChannelServerVersion: v2.9.99
```

Comment:
maxChannelServerVersion: v2.10.99 for v2.10

Reply:
Done!